### PR TITLE
fs-snapshot: Exclude mount points recursively

### PR DESCRIPTION
**Pull request by nicksmith53**
If `/home` is listed as excluded, exclude `/home/alice` and `/home/bob`

### CODE REVIEW

**Comment by james-antill** [comment]
The problem is that there's no way to unexclude. So if a user wants to exclude / but expects the mountpoint for /usr to be snapshotted ... it's not possible.

**Comment by nicksmith53** [comment]
Ah good point. OK what about adding another option in the config file for `exclude_recursive`

**Comment by james-antill** [comment]
Maybe better to have something like: exclude_glob = /home/*

If you'd prefer to just match prefixes then name it exclude_prefix (makes it more obvious that it's not walking the FS or something).

File: docs/yum-fs-snapshot.conf.5 [hunk 1]
```diff
@@ -21,6 +21,10 @@ utilizes configuration options in the form of
 This is a space delimited list of the mount points you do not wish to have
 snapshotted by this plugin.
 .SH OPTION
+.IP exclude_prefix
+This is a space delimited list of path prefixes to match mount points you
+do not want to have snapshotted by this plugin. (e.g. "/mnt")
```
Comment (dmnks) [inline docs/yum-fs-snapshot.conf.5]: I would maybe add here an example of what having "/mnt" there would actually do (ie. basically the example in your commit message).

**Review by dmnks — CHANGES_REQUESTED** [review]
Please see my note inline.

**Comment by nicksmith53** [comment]
Changed and rebased

**Comment by dmnks** [comment]
LGTM

**Comment by dmnks** [comment]
Thank you for the PR. However, please note that this project has been deprecated and is no longer maintained upstream. Please consider migrating to [DNF](https://github.com/rpm-software-management/dnf) which is the successor of YUM. That said, I’ll close this PR now.

**Pull request closed without merging**